tests/terraform/checks/resource/azure/example_AzureContainerInstanceEnvVarSecureValueType/main.tf

```diff
@@ -0,0 +1,151 @@
+# Variables declaration:
+
+variable "pud_default_var" {
+  default = "pud_default_value"
+}
+
+# Case 1: Pass: 'secure_environment_variables' exists in 'container' block
+
+resource "azurerm_container_group" "pass_1" {
+  name                = "pud_pass_1_container"
+  location            = var.pud_default_var
+  resource_group_name = var.pud_default_var
+  ip_address_type     = "Public"
+  dns_name_label      = "aci-label"
+  os_type             = "Linux"
+
+  container {
+    name   = "hello-world"
+    image  = "mcr.microsoft.com/azuredocs/aci-helloworld:latest"
+    cpu    = "0.5"
+    memory = "1.5"
+
+    ports {
+      port     = 443
+      protocol = "TCP"
+    }
+  }
+
+  container {
+    name   = "sidecar"
+    image  = "mcr.microsoft.com/azuredocs/aci-tutorial-sidecar"
+    cpu    = "0.5"
+    memory = "1.5"
+
+    secure_environment_variables = {
+      INIT_SECRET_VAR               = "secret_value"
```

Comment:
@praveen-panw Please add a skip comment because it won't pass tests

Reply:
Like this?:

    secure_environment_variables = {
      INIT_SECRET_VAR               = "secret_value"  # checkov:skip=CKV_SECRET_80 test secret
    }

Only for `secure_environment_variables` right?